exchange/exchange.go

```diff
@@ -176,6 +176,11 @@ func (e *exchange) HoldAuction(ctx context.Context, bidRequest *openrtb.BidReque
 	}
 
 	bidResponseExt := e.makeExtBidResponse(adapterBids, adapterExtra, bidRequest, debugInfo, errs)
+	if start != nil {
+		bidResponseExt.Prebid = &openrtb_ext.ExtResponsePrebid{
+			AuctionTimestamp: start.UnixNano() / 1e+6,
+		}
+	}
```

Comment:
Consider moving lines 179 to 183 inside `makeExtBidResponse`.

Reply:
I agree this code belongs in `makeExtBidResponse`. I wanted to avoid changing more signatures with the associated impact on unit tests... I'll take another look after #1570 settles.

Reply:
Hi @laurb9 friendly reminder that #1570 merged. No hurry though